cogs/utils/constants.py

```diff
@@ -21,10 +25,65 @@
     "&redirect_uri=https%3A%2F%2Fdiscord.gg%2FvP26dCy&scope=bot "
 )
 
+SUPPORT_SERVER = "https://discord.gg/CgH6Sj6"
+
 COMMAND_PREFIX = ">>"
 
 line_img_url = "https://cdn.discordapp.com/attachments/581139962611892229/692712698487767080/animated_line.gif"
 
+NEGATIVE_REPLIES = [
```

Comment:
```suggestion
# Magic 8-Ball responses
NEGATIVE_REPLIES = [
```

`NEGATIVE_REPLIES` alone isn't very clear we should either use a comment, as suggested or rename the variables to something like `BALL8_NEGATIVE_REPLIES`